### ISSUE TITLE
Import/Export

### ISSUE THREAD

**Issue by xbjfk** [CubeStuffs/CubeTime#7]
- [ ] cstimer
    - [ ] use api - post to https://cstimer.net/userdata.php with id= returns lzstring compressed json data (id is alphanumeric only)
    - [ ] use gdrive + wca oath for importing of cstimer solves
    - [ ] possibly sync seamlessly so cstimer can be used alongside, synced.
- [ ] chaotimer
    - [ ] use a desktop companion app with libimobiledevice to get data
- [ ] twisty timer
    - [ ] oath google drive
    - [ ] import "for backup" (with sessions) and not "for other timers"
- [ ] tell users to file a github issue if a timer is not listed

**Comment by pdtxie** [comment]
export to .txt / .csv
export images of stats page (?)

**Comment by feigaoxyz** [comment]
Export to csv

**Comment by pdtxie** [comment]
mentioned in https://github.com/CubeStuffs/CubeTime/issues/165

**Comment by afonsofrancof** [comment]
Any news on this? Even a basic csv export would be great. I could then manually modify it and import it into cstimer.

**Comment by brandonleon** [comment]
+1 for a CSV export.

**Comment by Danny132009** [comment]
Just wondering if there will be the addition of a csv export so I can import it into cstimer?

**Comment by pdtxie** [comment]
Yes, this is our top priority as it has also been requested through ko-fi! We have been very very busy the past year but we anticipate launching this feature by this month.

**Comment by mkmtrd** [comment]
Really waiting for import feature!

**Comment by xbjfk** [comment]
You can now export to CSV, ODS and csTimer formats. 
Import coming soon.

**Comment by ahmedshibly** [comment]
I have recently moved to cube time,CSV import would be really awesome.

**Comment by Skygames-yt** [comment]
When does the import feature come out?

**Comment by maxxorzz** [comment]
Any news on import/export/sync with cstimer? 

Its the only thing holding me back to switch fully to cubetime.

**Comment by M3rlin576** [comment]
Is the import feature in the works? I'm eager to switch over to cubetime

**Comment by giankdiluvi** [comment]
For anyone who is struggling to export their data using an iPad in dark mode, take a look at #241. The session manager features are hard to see in dark mode, but switching to light mode resolves this. From there, you should be able to export your data.
